tests/acceptance/blocks/verticalresults.js

```diff
@@ -32,19 +35,47 @@ class VerticalResults {
   }
 
   /**
-   * Gets a selector for the card associated with the currently focused map pin.
-   * @returns {Selector}
+   * Returns the number of results from the VerticalResultsCount-start.
+   * @returns {Promise<number>}
    */
-  getFocusedCard () {
-    return this._focusedCard;
+   async getResultsCountStart () {
+    const countText = await this._resultsCountStart.innerText;
+    return Number.parseInt(countText);
   }
 
   /**
-   * Gets a selector for the results div.
-   * @returns {Selector}
+   * Returns true if a card from the vertical results is focused
+   * @returns {Promise<boolean>}
    */
-  getResults () {
-    return this._selector;
+  async isCardFocused () {
+    return this._focusedCard.exists;
+  }
+
+  /**
+   * Clicks the first result card
+   */
+  async clickFirstCard () {
```

Comment:
Could we make a general `clickCard(cardIndex)` method? This would then be a special case of that. I could see us wanting the more general method for future tests.

Reply:
sure